src/base/settingsstorage.cpp

```diff
@@ -125,6 +125,9 @@ namespace
             {"BitTorrent/Session/MaxHalfOpenConnections", "Preferences/Connection/MaxHalfOpenConnec"},
             {"BitTorrent/Session/uTPEnabled", "Preferences/Bittorrent/uTP"},
             {"BitTorrent/Session/uTPRateLimited", "Preferences/Bittorrent/uTP_rate_limited"},
+            {"BitTorrent/Session/peerTurnover", "Preferences/Bittorrent/peerTurnover"},
+            {"BitTorrent/Session/peerTurnoverCutoff", "Preferences/Bittorrent/peerTurnoverCutoff"},
+            {"BitTorrent/Session/peerTurnoverInterval", "Preferences/Bittorrent/peerTurnoverInterval"},
```

Comment:
Is this part necessary?

Reply:
>Is this part necessary?

They are redundant.

Reply:
I have no idea - I just added new similar code wherever other advanced settings had code. If it is not needed I can drop it.